### ISSUE TITLE
[P3.1][Sourcing][Accessibility][ATL testing][Multi-input]: multi-input on searching ,  the results count not read by JAWS for combobox 

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10621]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
multi-input on searching ,  the results count not read by JAWS for combobox 
Expectation: when user searches some text in multi-input and the list is opened with some number of list items , the list items count is not announced for matching results by screenreader

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

ng: 15
platform: 0.43.x

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
replicable on https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/multi-input#simple

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
reported by ATL team

**Comment by droshev** [comment]
We need to announce the number of the matching items. The issue can be processed with this [one](https://github.com/SAP/fundamental-ngx/issues/10622)

**Comment by mikerodonnell89** [comment]
fixed with #10622
